src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisInMemoryObserver.java

```diff
@@ -0,0 +1,29 @@
+package org.gridsuite.sensitivityanalysis.server.service;
+
+import com.powsybl.sensitivity.SensitivityAnalysisResult;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.observation.ObservationRegistry;
+import lombok.NonNull;
+import org.gridsuite.sensitivityanalysis.server.computation.service.AbstractComputationObserver;
+import org.gridsuite.sensitivityanalysis.server.dto.SensitivityAnalysisInputData;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SensitivityAnalysisInMemoryObserver extends AbstractComputationObserver<SensitivityAnalysisResult, SensitivityAnalysisInputData> {
+    private static final String COMPUTATION_TYPE = "sensiInMemory";
+
```

Comment:
Had to create a secondary observer to make the type work => should I rework the Observer class to make it possible tu merge the 2 sensi observer classes ?